tests/test_rsa_keys.py

```diff
@@ -158,28 +159,39 @@ def test_verify_rsa_signature(self):
 
 
   def test_verify_rsa_pss_sha256(self):
-    global public_rsa
-    global private_rsa
-
     rsa_scheme = 'rsassa-pss-sha256'
-    data = 'The quick brown fox jumps over the lazy dog'.encode('utf-8')
+    data = 'Some say the bigger the salt, the more provable the security'.encode('utf-8')
 
     private_key_object = load_pem_private_key(private_rsa.encode('utf-8'),
         password=None, backend=default_backend())
+    public_key_object = load_pem_public_key(public_rsa.encode('utf-8'),
+        backend=default_backend())
     digest_obj = securesystemslib.hash.digest_from_rsa_scheme(rsa_scheme,
         'pyca_crypto')
     max_salt_length = padding.calculate_max_pss_salt_length(private_key_object, digest_obj)
 
+    # Check that every valid salt length value can be automatically verified.
     for salt_length in range(max_salt_length+1):
       signature = private_key_object.sign(
           data, padding.PSS(mgf=padding.MGF1(digest_obj.algorithm),
           salt_length=salt_length), digest_obj.algorithm)
 
-      valid_signature = \
-        securesystemslib.rsa_keys.verify_rsa_signature(signature,
-          rsa_scheme, public_rsa, data)
-
-      self.assertEqual(True, valid_signature)
+      # Current behaviour: automatic inferring of salt length.
+      self.assertTrue(securesystemslib.rsa_keys.verify_rsa_signature(signature,
+          rsa_scheme, public_rsa, data))
+
+      # Previous behaviour: generally wrong assumption about salt length.
```

Comment:
What's the purpose of the two tests below? They seem to test `cryptography` API and not our code. Or is this a regression test that accepts a regression? :D

Reply:
I think my thinking was to show that previously we would have failed, and now we don't, although that's not clear by looking at this. More importantly, I need to add tests to show that: (1) _previous_ signatures are still automatically verifiable, (2) new signatures are automatically verifiable (I think we have already done that here), and (3) new signatures from _other_ platforms (e.g., Vault) are automatically verifiable.